vroxy.py

```diff
@@ -65,9 +68,17 @@ def extractExpiry(self) -> float:
 config["server"] = {
     "host": os.getenv("VROXY_HOST", "0.0.0.0"),
     "port": os.getenv("PORT", "8008"),
+    "whitelist": os.getenv("VROXY_WHITELIST", ""),
 }
 if path.isfile(path.join(path.dirname(__file__), "settings.ini")): config.read(path.join(path.dirname(__file__), "settings.ini"))
 
+domain_whitelist: Optional[DomainWhitelist] = None
+
+if wl_path := config["server"]["whitelist"]:
+    wl_path = Path(wl_path)
+    if wl_path.exists():
+        domain_whitelist = load_list(str(wl_path))
```

Comment:
Actually this will fail silently in a nasty way. Let's just blow up if the user specifies a whitelist that doesn't exist.

```suggestion
    domain_whitelist = load_list(wl_path)
```